src/templates/common/breadcrumbs/index.tsx

```diff
@@ -0,0 +1,94 @@
+import {
+  transformBreadcrumbs,
+  deriveLastBreadcrumbFromPath,
+  deriveLcBreadcrumbs,
+  filterInvalidCrumbs,
+} from '@/templates/globals/util/breadcrumbUtils'
+import { BreadcrumbItem } from '@/types/dataTypes/drupal/field_type'
+
+interface BreadcrumbProps {
+  breadcrumbs?: BreadcrumbItem[]
+  breadcrumbTitle?: string
+  disableAnalytics?: boolean
+  deriveBreadcrumbsFromUrl?: boolean
+  replaceLastItem?: boolean
+  constructLcBreadcrumbs?: boolean
+  titleInclude?: boolean
+  hideHomeBreadcrumb?: boolean
+  customCrumbHomeText?: string
+  entityPath?: string
+}
+const Breadcrumbs = ({
+  breadcrumbs,
+  entityPath,
+  breadcrumbTitle,
+  disableAnalytics,
+  deriveBreadcrumbsFromUrl,
+  replaceLastItem,
+  constructLcBreadcrumbs,
+  titleInclude,
+  hideHomeBreadcrumb,
+  customCrumbHomeText,
```

Comment:
This relationship is a bit interesting to me. 
### Scenario 1 (default, makes sense)
```
default
```
Home > Path > To > Page

### Scenario 2 (makes sense)
```
hideHomeBreadcrumb: true, 
```
Path > To > Page

### Scenario 3 (kind of makes sense, depending how you look at it)
```
hideHomeBreadcrumb: true, 
customCrumbHomeText: 'MyCustonHomeCrumb',
```
MyCustomHomeCrumb > Path > To > Page

### Scenario 4 (feels very odd to me given current naming/structure)
```
customCrumbHomeText: 'MyCustonHomeCrumb',
```
Home > Path > To > Page

Reply:
I considered this a bit as well, and maybe wrongly left it as is.   Looking again, I think its more logical that `if customCrumbHomeText -> set hideHomeBreadcrumb to true` so that the consumer doesnt need to be aware of that relationship at all.

Reply:
I think this largely makes sense, though I might put it slightly differently. To me, `hideHomeBreadcrumb` should be the prop with higher authority. It should default to `false`, but, if it's set to `true`, there should be no home breadcrumb shown, regardless of whether a custom one is passed in. If it's not set (or explicitly set to `false`), its contents should then fall to whether a custom value is passed in. If so, use the custom value. If not, "Home".

### Default
```
default
```
Home > Path > To > Page

### Hide home
```
hideHomeBreadcrumb: true, 
customCrumbHomeText: 'MyCustonHomeCrumb',  // this is ignored
```
Path > To > Page

### Replace home
```
customCrumbHomeText: 'MyCustonHomeCrumb',
```
MyCustonHomeCrumb > Path > To > Page

Reply:
fixed with new if/else logic

Reply:
I'm thinking that if customCrumbText is passed in at all, then the user wants to see it displayed.  I can see a case for either, but will keep the updated if/else logic for now and can update later if found confusing